packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -47,6 +47,15 @@
                     </f-button>
                 </slot>
 
+                <component
+                    :is="titleHtmlTag"
+                    v-if="title"
+                    :class="$style['c-megaModal-title']"
+                    data-test-id="mega-modal-title"
+                >
+                    {{ title }}
```

Comment:
On menu we use a `slot`, here (`<slot name="heading" />`) rather than a prop, as we pass some additional markup to allow for things like sticky headers etc.

Reply:
I'll look into it now, thanks!

Reply:
@DamianMullins you can still define your own heading without using the prop. It's not a breaking change, your code will still work. This is for situations when you have a standard heading.  I think that the way it works now covers the situation on menu where you need additional markup and custom styles.

I think that ideally, we should have the sticky headers feature in `f-mega-modal`. 

What do you think?